docs/api/web-frame-main.md

```diff
@@ -66,6 +66,15 @@ These methods can be accessed from the `webFrameMain` module:
 Returns `WebFrameMain | undefined` - A frame with the given process and routing IDs,
 or `undefined` if there is no WebFrameMain associated with the given IDs.
 
+### `webFrameMain.fromFrameToken(processId, frameToken)`
+
+* `processId` Integer - An `Integer` representing the internal ID of the process which owns the frame.
+* `frameToken` string - A `string` representing the unique frame token in the
+  current renderer process.
+
+Returns `WebFrameMain | undefined` - A frame with the given process and frame token,
```

Comment:
Is there a reason that `WebFrameMain` returns `undefined` but `WebFrame` returns `null`? Or is that just an artifact of the APIs evolving independently.

Reply:
The latter. Definitely open to changing this at some point later on 😄